### ISSUE TITLE
Odd behavior when invoking Temp Map / Markers dialog

### ISSUE THREAD

**Issue by windowsrefund** [rncbc/qtractor#492]
play-head is currently on measure 34 where I've carved out a nice little section for a meter change. OK cool, open the `Tempo Map / Markers` dialog where I know I'll specify the measure to adjust....

Boom! The focus of the Arranger view "snaps" back to the beginning of the track and now I can't recall what measure I was interested in tweaking (it's now out of view). Worth noting perhaps is that `Follow play-head` is enabled.

Looking at the fact `1.1.000` is hard-coded in the dialog, I'm guessing this is actually working correctly since I'm guessing that first entry is also sort-of "highlighted" by default which probably triggers some kind of "go to" functionality. Though, I'd argue I never actually focused that record and I do see I can highlight it with the mouse. So, I think it's working according to design but that design pretty much "jumps the gun" on what we're actually trying to do. 

If possible, it'd be great if the dialog didn't make the "let's navigate to 1.1.000" assumption and just waited for the user to use the tool?

Not a big deal but interesting.

**Comment by rncbc** [comment]
yes, if summoned from the menu View > Tempo Map / Markers... the dialog makes it to the first node location, which is, no other than absolute t=0 ie. BBT=1.1.000;, but if you double-click on the top time-ruler header it goes up at the pointed location.

question is, what location should it point when raised from the menu?
a. play-head (red cursor-line);
b. edit-head (left blue cursor-line);
c. or somewhere in the current visible canvas viewport (maybe half-way center)?

**Comment by windowsrefund** [comment]
Oh interesting! I never knew the tool could be invoked by clicking on the timeline. This is very nice!

As for your question, I'd think if it had to navigate automatically, A would be the move. I could probably even back that opinion up by pointing out another menu operation; `Clip/New` which renders the new clip at the location of play-head. In general, adhering to that pattern would translate to something in terms of consistency.

**Comment by windowsrefund** [comment]
Thanks for that. This is working as expected. Very nice.
